programmers/5일차/제일-작은수-제거하기.js

```diff
@@ -0,0 +1,27 @@
+function solution(arr) {
+    const minNum = Math.min.apply(null,arr);
```

Comment:
```suggestion
    const minNum = Math.min(...arr);
```
이렇게도  최솟값을 구할 수 있어요  ...은 전개 연산자라는 문법이에요  개발할때도 자주 사용되는 문법이니 공부해보시면 좋을 것 같습니다